core/src/main/resources/defaults/clustertemplates/7.2.8/aws/dataengineering-ha.json

```diff
@@ -103,7 +103,31 @@
       {
         "nodeCount": 2,
         "name": "master",
-        "type": "GATEWAY",
+        "type": "CORE",
```

Comment:
@sidseth One good thing about the HA blueprint is it has the Knox gateways added explicitly in the blueprint. So Knox will be HA after this change. But what will be the effect of this change is that the Raz will not be HA, which I was anticipating to be HA. Adding @doktoric. If we move to a model where Raz can be added to an instance group with at least two instances then we can achieve RAZ HA, and we can track that separately. I know there is an email thread about this. So will post it there as well.

Reply:
7.2.8 was the only template which had to be changed for this, all others templates master was CORE only. This was the approach discussed on call with you and ganesh.  Maybe we can increase nodeCount of the new manager hostgrup with only CM to 2, so that RAZ is also HA ?  cc @cegganesh84 , @sidseth

Reply:
Ok. There's also https://github.com/hortonworks/cloudbreak/pull/10266/ .
End of the day - I think the 7.2.8, 7.2.9, and 7.2.10 templates need to reach the same state.

Reply:
Commit after renaming the group, and then a separate change to rationalize the 3 versions?

Reply:
Raz HA is taken care of by #10290